styles.css

```diff
@@ -0,0 +1,542 @@
+}
+
+.main-program {
+  padding-top: 3vw;
+  padding-bottom: 3vw;
+  width: 100%;
+}
+
+.title {
+  color: white;
+  font-size: 15px;
+  padding-bottom: 0.5vw;
+  background: url(../capstoneimgs/title_underline.png) bottom no-repeat;
+}
+
+/* #header a {
+  text-decoration: none;
+  color: #ff5438;
+} */
+
+h3 {
+  font-size: 18px;
+  color: #ff5438;
+}
+
+.program-cards {
+  display: grid;
+  grid-template-columns: 100px 100px auto;
+  justify-content: flex-start;
+  align-items: center;
+  color: white;
+  padding: 0.5em;
+  background-color: rgba(65, 66, 70, 0.4);
+}
+
+.join-inti {
+  padding: 1.5em;
+  margin: 4em;
+  width: 75%;
+  color: white;
+  background-color: #ff5438;
+}
+
+.desktop-see-program {
+  display: none;
+}
+
+.main-performers {
+  height: auto;
+}
+
+#perTitle {
+  text-align: center;
+  color: black;
+  font-size: 15px;
+  margin-bottom: 0;
+}
+
+.red-line {
+  width: 100%;
+  height: auto;
+  background: url('../capstoneimgs/title_underline.png') no-repeat;
+}
+
+.container-performers {
+  padding: 1rem 6rem 6rem 6rem;
+}
+
+#all-performers {
+  flex-wrap: wrap;
+  margin-bottom: 0.5rem;
+  padding-inline: 3em;
+  height: 650px;
+  overflow: hidden;
+}
+
+.performer-card {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 2em;
+}
+
+.checkerbg {
+  background: url('../capstoneimgs/checkboardimg.jpg') top left no-repeat;
+  background-size: contain;
+  justify-self: center;
+  padding: 0.7em 0 0 0.7em;
+  width: 3.5em;
+  height: 3.5em;
+}
+
+.band-detail {
+  display: flex;
+  flex-direction: column;
+  margin-top: 9vw;
+  margin-left: 12vw;
+}
+
+.d-flex {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.band-brief-info {
+  font-size: 16px;
+  padding-bottom: 10px;
+  color: #ff5438;
+  background: url(../capstoneimgs/title_underlinegray.svg) bottom left no-repeat;
+}
+
+.artist-intro {
+  font-size: 16px;
+  padding-top: 15px;
+  padding-bottom: 10px;
+}
+
+.more-less {
+  display: flex;
+  justify-content: center;
+  align-self: center;
+  margin: 4em;
+  width: 75%;
+  color: black;
+  height: auto;
+  background-color: white;
+  border: 1px solid grey;
+  padding: 0.8em;
+}
+
+.more-less::after {
+  display: flex;
+  transition: 0.2s;
+  font-size: 22px;
+  text-align: center;
+  align-self: center;
+  align-items: center;
+}
+
+.more-lessExpand {
+  height: auto;
+  overflow: visible !important;
+}
+
+.titlePartners {
+  display: none;
+}
+
+#sponsors {
+  display: none;
+}
+
+.desktop-footer {
+  display: flex;
+  flex-direction: row;
+  background-color: gray;
+  width: 100%;
+  height: auto;
+  align-items: end;
+}
+
+.footer-p {
+  width: 100%;
+  color: #414246;
+  align-self: center;
+}
+
+/****** DESKTOP STYLES ********************************/
+
+@media screen and (min-width: 768px) {
+  body {
+    overflow-x: hidden;
+  }
+
+  #header-nav-bar {
+    display: none;
+  }
+
+  .session-container {
+    display: flex !important;
```

Comment:
[OPTIONAL] You used !important for 3 times, I can't say it is overuse. But please try to fix your CSS code to use less !important declaration.

Reply:
Sure! Thanks for the suggestion, some of the !important I've used is for functionality like the more-less button! I will try not to use them as much. :)